docs/semconv/database.md

```diff
@@ -261,6 +265,36 @@ This metric is [optional](https://github.com/open-telemetry/semantic-conventions
 |--------------------------|-----------------|------------------|----------------------------------------------------------------------------------------------|
 | `db.overflow.user.count` | UpDownCounter   | `{overflowUser}` | Number of times a User thread attempted to acquire a lock when no locks were available.      |
 
+### Metric: `db.cache.read.ratio`
+This metric is [optional](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metric-requirement-level.md#opt-in).
+
+| Name                  | Instrument Type   | Units (UCUM)       | Description                                                                              |
+|-----------------------|-------------------|--------------------|------------------------------------------------------------------------------------------|
+| `db.cache.read.ratio` | UpDownCounter     | `{cacheReadRatio}` | Percentage of page reads for this buffer pool that were satisfied by a cached page image |
+
+
+## Metric: `db.cache.write.ratio`
+This metric is [optional](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metric-requirement-level.md#opt-in).
+
+| Name                   | Instrument Type   | Units (UCUM)        | Description                                                                               |
+|------------------------|-------------------|---------------------|-------------------------------------------------------------------------------------------|
+| `db.cache.write.ratio` | UpDownCounter     | `{cacheWriteRatio}` | Percentage of page Writes for this buffer pool that were satisfied by a cached page image |
+
+### Metric: `db.lock.waits`
```

Comment:
Do you think that would it be easier for other users to read by arranging each metrics details section in the order in which the metrics appear in the catalog you listed there:
```
  [Metric `db.lock.waits`](#metric-dblockwaits)
  [Metric `db.cache.read.ratio`](#metric-dbcachereadratio)
  [Metric `db.cache.write.ratio`](#metric-dbcachewriteratio)
  [Metric `db.lru.writes`](#metric-dblruwrites)
```